### PR TITLE
MESSAGEix manuscript reference update

### PR DESCRIPTION
**Pull request by danielhuppmann**
This PR updates the references in the documentation, notice, etc. to the published version of the MESSAGEix manuscript and to the latest version of the MESSAGEix South Africa manuscript.

### CODE REVIEW

File: doc/source/bibs/main.bib [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
-	Title = {{The MESSAGEix Integrated Assessment Model and the ix modeling platform}},
-	Url = {https://pure.iiasa.ac.at/15157/},
-	Year = {submitted}}
+@article{huppmann_messageix_2018,
+   author = {Huppmann, Daniel and Gidden, Matthew and Fricko, Oliver and
+             Kolp, Peter and Orthofer, Clara and Pimmer, Michael and
+             Kushin, Nikolay and Vinca, Adriano and Mastrucci, Alessio and
+             Riahi, Keywan and Krey, Volker},
+   title = {{The MESSAGEix Integrated Assessment Model and the ix modeling platform (ixmp): An open framework for integrated and cross-cutting analysis of energy, climate, the environment, and sustainable development}},
+   journal = {Environmental Modelling & Software},
```
Comment (khaeru) [inline doc/source/bibs/main.bib]: `&` in LaTeX needs to be escaped: `\&`

Comment (danielhuppmann) [reply doc/source/bibs/main.bib]: thanks @khaeru

**Comment by danielhuppmann** [comment]
@gidden left for an extended Christmas vacation, let's not bother him with that

**Comment by khaeru** [comment]
This now seems ready! Please go ahead and merge.

**Comment by danielhuppmann** [comment]
merging as approved by @khaeru

**Pull request merged**